src/hyperion/experiment_plans/rotation_scan_plan.py

```diff
@@ -199,6 +200,13 @@ def rotation_scan_plan(
 
     yield from arm_zebra(zebra)
 
+    # Check topup gate
+    yield from check_topup_and_wait_if_necessary(
+        i03.synchrotron(),
+        detector_params,
+        ops_time=10.0,  # Additional time to account for rotation, is s
```

Comment:
I think this should come from some kind of property/parameter long-term? It may be different across different beamlines which each eventually use this plan. Or alternatively scientists/developers may need to adjust this periodically. i.e. I don't think this should be hardcoded. Can be spun into an issue if you prefer.